app/views/visitors/index.html.erb

```diff
@@ -1 +1,2 @@
-<h3>Welcome</h3>
+<h3>Homes England</h3>
+<%= button_to 'Login', 'auth/auth0', method: :post %>
```

Comment:
I don't think it matters much at this point but we should probably use the language recommended by the[ GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#sign-in-or-log-in). eg "Sign in", "Sign out" etc